tools/releasing/create_binary_release.sh

```diff
@@ -109,14 +112,30 @@ make_python_release() {
 
   cp ${pyflink_actual_name} "${RELEASE_DIR}/${pyflink_release_name}"
 
+  # RM need to move the downloaded wheel packages from Azure CI to the directory of flink-python/dist manually.
+  for wheel_file in *.whl; do
+    if [[ ! ${wheel_file} =~ ^apache_flink-$PYFLINK_VERSION- ]]; then
```

Comment:
So the name of the wheel package is `apache_flink-version.whl` instead of `apache-flink-version.whl`?